src/routes/RepositoryDetails/Tags/SecurityDetails.tsx

```diff
@@ -27,9 +73,63 @@ export default function SecurityDetails(props: SecurityDetailsProps) {
   if (props.arch) {
     queryParams.set('arch', props.arch);
   }
+
+  if (loading && props.variant === Variant.condensed) {
+    return <Skeleton />;
+  } else if (loading) {
+    return <Spinner isSVG />;
+  }
+
+  if (vulnCount.size === 0) {
+    return (
+      <Link
+        to={getTagDetailPath(props.org, props.repo, props.tag, queryParams)}
+      >
+        <CheckCircleIcon color="green" />
+        <span>None Detected</span>
+      </Link>
+    );
+  }
+
+  if (props.variant === Variant.condensed) {
+    const highestSeverity: VulnerabilitySeverity = getHighestSeverity();
+    return (
+      <Link
+        to={getTagDetailPath(props.org, props.repo, props.tag, queryParams)}
+      >
+        <div>
+          <ExclamationTriangleIcon
+            color={getSeverityColor(highestSeverity)}
+            style={{
+              marginRight: '5px',
+            }}
+          />
+          <span>
+            <b>{vulnCount.get(highestSeverity)}</b> {highestSeverity.toString()}
+          </span>
+        </div>
+      </Link>
+    );
+  }
+
+  const counts = severityOrder
```

Comment:
There's a lot of overlap between this logic and the one I have for Packages. Let's revisit this once that is merged

Reply:
Sounds good